src/docs/sdk/features.mdx

```diff
@@ -75,9 +75,36 @@ Ability to get the ID of the last event sent. Event IDs are useful for correlati
 
 ## User Feedback
 
-On user-facing platforms such as mobile or the browser this means first class support for requesting user feedback when an error occurs.
+For all SDKs, it is strongly recommended to send the user feedback as an [envelope](/sdk/envelopes/#user-feedback). Alternatively, the SDKs can
+use the [user feedback endpoint](https://docs.sentry.io/api/projects/submit-user-feedback/), which is not recommended.
 
-On backend platforms, SDKs should document how to use the last event ID to prompt the user for feedback themselves.
+### User Facing Platforms
+On user-facing platforms such as mobile, desktop, or browser this means first-class support for requesting user feedback when an error or crash occurs.
+To see some examples of the API check out the user-facing docs for [Apple](https://docs.sentry.io/platforms/apple/enriching-events/user-feedback/) and
+[Java](https://docs.sentry.io/platforms/java/enriching-events/user-feedback/).
+
+On mobile and desktop, it is common to prompt the user for feedback after a crash happened on the previous run of the application. Therefore the SDKs should
+implement the `onCrashedLastRun` callback on the options. This callback gets called shortly after the initialization of the SDK when the last program execution
+terminated with a crash. The SDK should execute the callback only once during the entire run of the program to avoid multiple callbacks if there are multiple 
+crash events to send.
+
+### Backend Platforms
+
+On backend platforms, SDKs should document how to use the last event ID to prompt the user for feedback themselves. 
```

Comment:
L: maybe we link the `last event ID` to the docs? I guess it might be described in the unified API docs